language/move-vm/types/src/data_store.rs

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) The Libra Core Contributors
```

Comment:
this is the trait and the data store API that has to be provided to the Move VM.
Maybe `RemoteCache` could be moved here?

Reply:
I feel like the remote cache isn't a concept that needs to be exposed to the Move VM. To me it's specific to Libra and therefore should stay in the Libra/Move adapter layer.